pkg/kubelet/dockershim/docker_container.go

```diff
@@ -347,7 +347,7 @@ func (ds *dockerService) ContainerStatus(_ context.Context, req *runtimeapi.Cont
 	// Convert the image id to a pullable id.
 	ir, err := ds.client.InspectImageByID(r.Image)
```

Comment:
Could one possible option be checking that an image with the `id` exists? If not, we could return without error.

Reply:
But the other `InspectImageByID` err  can also case kubelet stuch